app/controllers/audits/audits_controller.rb

```diff
@@ -59,5 +63,26 @@ def audit_params
           )
         )
     end
+
```

Comment:
The controller seems to be doing too much. Could you think on ways to move this code out of here?